game/graphics/opengl_renderer/shaders/direct_basic_textured_tcc0.frag

```diff
@@ -1,18 +0,0 @@
-#version 330 core
```

Comment:
I think you need to remove these from `Shader.cpp` and `Shader.h`

Reply:
removed them right before you said it :)